src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -82,6 +82,35 @@ echo "Running '{{ PackageInstallCommand }}'..."
 echo
 {{ PackageInstallCommand }}
 
+{{ if AppInsightsInjectCommand | IsNotBlank }}
+echo
+echo "Preparing environment to set up Application Insights..."
+echo "Running '{{ AppInsightsInjectCommand }}'..."
+echo
+{{ AppInsightsInjectCommand }}
+echo
+echo "Creating Application Insights loader file in '$DESTINATION_DIR'"
+echo "// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+// Created by Oryx
+try 
+{
+	var appInsights = require('{{ AppInsightsPackageName }}');
+	if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY) {
+
+        appInsights
+            .setup()
+            .start();
+	}
+}catch (e) {
+        console.log(e);
```

Comment:
Write some message to indicate what happened, e.g. "Application Insights could not be automatically configured for this application."